File: src/kyu_6/triple_trouble_1/csharp/triple_trouble_1_test.cs
using NUnit.Framework;
using System;

[TestFixture]
public class Tests
{
  [Test]
  [TestCase(451999277, 41177722899, ExpectedResult=1)]
  [TestCase(1222345, 12345, ExpectedResult=0)]
  [TestCase(12345, 12345, ExpectedResult=0)]
  [TestCase(666789, 12345667, ExpectedResult=1)]
  [TestCase(10560002, 100, ExpectedResult=1)]
  [TestCase(1112, 122, ExpectedResult=0)]
  public static int FixedTest(long s1, long s2)
  {
    return Kata.TripleDouble(s1, s2);
  }
}

File: src/kyu_6/triple_trouble_1/python/triple_trouble_1_test.py
from solution import triple_double
import codewars_test as test

@test.describe("Triple trouble")
def triple_trouble():

    @test.it("Sample tests")
    def sample_tests():
        test.assert_equals(triple_double(451999277, 41177722899), 1)
        test.assert_equals(triple_double(1222345, 12345), 0)
        test.assert_equals(triple_double(12345, 12345), 0)
        test.assert_equals(triple_double(666789, 12345667), 1)
        test.assert_equals(triple_double(10560002, 100), 1)
        test.assert_equals(triple_double(1112, 122), 0)